terra_layer/models.py

```diff
@@ -216,6 +216,33 @@ def save(self, **kwargs):
     def __str__(self):
         return f"Layer({self.id}) - {self.name}"
 
+    @transaction.atomic()
+    def replace_source(self, source, fields_matches={}):
```

Comment:
You can't use a dict a default value. See https://pythonconquerstheuniverse.wordpress.com/category/python-gotchas/

Reply:
Fixed.